src/pages/Items/Product.jsx

```diff
@@ -0,0 +1,368 @@
+import styled from "@emotion/styled";
+import Header from "./Header";
+import InquiryItem from "./InquiryItem";
+import { useEffect, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+
+// 기존 스타일 유지
+const Container = styled.div`
+  width: 100%;
+  max-width: 1200px;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 64px;
+  margin: 0 auto;
+  padding: 94px 16px 0;
+`;
+
+const ProductWrapper = styled.div`
+  width: 100%;
+  height: auto;
+  display: flex;
+  gap: 20px;
+
+  @media (max-width: 768px) {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto; /* 🔧 오타 수정: margi → margin */
+    align-items: center;
+  }
+`;
+
+const ProductImage = styled.div`
+  width: 486px;
+  height: 486px;
+  background-color: #f3f4f6;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+  border-radius: 8px;
+
+  img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  @media (max-width: 768px) {
+    width: 100%;
+    max-width: 100%;
+    height: auto; /* 높이 자동으로 */
+    aspect-ratio: 1 / 1; /* ✅ 정사각형 비율 유지 */
+    margin-bottom: 20px;
+
+    img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+`;
+
+const ProductDescription = styled.div`
+  width: 680px;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+
+  @media (max-width: 768px) {
+    width: 100%;
+    max-width: 486px;
+    padding: 0 16px; /* ✅ 모바일 대응 */
+  }
+`;
+
+const ProductName = styled.h2`
+  font-size: 24px;
+  font-weight: bold;
+`;
+
+const ProductPrice = styled.h3`
+  font-size: 20px;
+  font-weight: 500;
+`;
+
+const SectionTitle = styled.h4`
+  font-size: 16px;
+  font-weight: bold;
+`;
+
+const ProductText = styled.p`
+  font-size: 14px;
+  color: #4b5563;
+`;
+
+const TagList = styled.div`
+  display: flex;
+  gap: 8px;
+`;
+
+const BottomMeta = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 32px;
+`;
+
+const WriterInfo = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`;
+
+const WriterAvatar = styled.img`
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+`;
+
+const WriterText = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`;
+
+const WriterName = styled.div`
+  font-size: 14px;
+  font-weight: 500;
+`;
+
+const CreatedDate = styled.div`
+  font-size: 12px;
+  color: #9ca3af;
+`;
+
+const Heart = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  cursor: pointer;
+  font-size: 14px;
+  color: #ef4444;
+  font-weight: bold;
+`;
+const Tag = styled.span`
+  font-size: 12px;
+  background-color: #f3f4f6;
+  padding: 4px 8px;
+  border-radius: 8px;
+`;
+
+const AddButton = styled.button`
+  width: 74px;
+  height: 42px;
+  border-radius: 8px;
+  background-color: ${({ disabled }) => (disabled ? "#9ca3af" : "#3692FF")};
```

Comment:
💊 제안
이런 경우 css의 가상 클래스를 사용해서 css 작성하시는 것이 더 적절해보입니다.
```suggestion
  background-color: #3692FF;
  &:disabled {
    background-color: #9ca3af;
  }
```